alacritty/src/display/window.rs

```diff
@@ -186,9 +190,29 @@ impl Window {
         let is_wayland = false;
 
         let windowed_context =
-            create_gl_window(window_builder.clone(), event_loop, false, !is_wayland, size)
+            create_gl_window(window_builder.clone(), event_loop, false, !is_wayland, false, size)
+                .or_else(|_| {
+                    create_gl_window(
+                        window_builder.clone(),
+                        event_loop,
+                        true,
+                        !is_wayland,
+                        false,
+                        size,
+                    )
+                })
+                .or_else(|_| {
+                    create_gl_window(
+                        window_builder.clone(),
+                        event_loop,
+                        false,
+                        !is_wayland,
+                        true,
+                        size,
+                    )
```

Comment:
The entire point of this is to act as a fallback in case one doesn't work, combining both srgb and 10-bit mode in the fallback doesn't sound like a good idea. We're having two branches here for a possibility of 4 different setups.

Reply:
Hence my comment about looking for ideas on how to do it better.

Reply:
With the color format the issue is that it might be different for every monitor. I'm not sure if depth can be defined on a per-monitor level but generally we should try to figure this out in advance.

Reply:
Glutin doesn't allow to do this now unfortunately. I'd suggest to fallback to srgb + 10 bit in the end.